src/include/migraphx/op/log2.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015-2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2015-2024 Advanced Micro Devices, Inc. All rights reserved.
```
CI probably failing due to this